### DISCUSSION TITLE
关于音频播放器的问题

### DISCUSSION THREAD

**Discussion by grace1533** [AlistGo/alist#5841] [Q&A] [answered]
在打开ALIS后播放音频都是随机的，如何用代码变成启动后就是顺序播放？
![Uploading 360截图20240108140307905.jpg…]()

**Comment by grace1533** [comment]
/
![360截图20240108140307905](https://github.com/alist-org/alist/assets/134708276/e386de1c-9f7c-4ea0-9199-9c1be4804655)

**Comment by anwen-anyi** [comment, marked as answer]
请更新版本 v3.29.0 就修复了
https://github.com/alist-org/alist-web/releases/tag/3.29.0
![image](https://github.com/alist-org/alist/assets/56105412/9a7d689a-25b3-44d7-af99-c14efa3759f5)


查看示例：
https://github.com/alist-org/alist/assets/56105412/30b60370-6e60-44b9-af36-2e2b4354cfef
